components/layout/display_list_builder.rs

```diff
@@ -1072,15 +1072,17 @@ impl FragmentDisplayListBuilding for Fragment {
                                                     style: &ServoComputedValues,
                                                     bounds: &Rect<Au>,
                                                     clip: &ClippingRegion) {
+        use style::values::Either;
+
         let width = style.get_outline().outline_width;
         if width == Au(0) {
             return
         }
 
-        let outline_style = style.get_outline().outline_style;
-        if outline_style == border_style::T::none {
-            return
-        }
+        let outline_style = match style.get_outline().outline_style {
+            Either::First(_auto) => border_style::T::none,
+            Either::Second(border_style) => border_style,
```

Comment:
Note that we should be returning in case of `border-style: none`. Also, since we don't handle `auto` right now, let's drop a `FIXME` comment saying that `outline-style: auto` should be handled.